components/core/email.conf

```diff
@@ -0,0 +1 @@
+{"username": "bassa.ucsc@gmail.com", "password": "bassa.123"}
```

Comment:
No. Nononono. Never ever store config within source, every major os gives you facilities to enable per user conf, even "~" resolvesv to user dir in windows too these days i guess please use them